File: Loenn/triggers/MultiMusicTrigger.lua
local MultiMusicTrigger = {}

MultiMusicTrigger.name = "XaphanHelper/MultiMusicTrigger"
MultiMusicTrigger.fieldOrder = {
    "x", "y", "width", "height", "flagA", "flagB", "trackNone", "trackA", "trackB", "trackBoth", "removeWhenOutside"
}
MultiMusicTrigger.placements = {
    name = "MultiMusicTrigger",
    data = {
        flagA = "",
        flagB = "",
        trackNone = "",
        trackA = "",
        trackB= "",
        trackBoth = "",
        removeWhenOutside = false
    }
}

return MultiMusicTrigger

File: Ahorn/triggers/MultiMusicTrigger.jl
module XaphanHelperMultiMusicTrigger

using ..Ahorn, Maple

@mapdef Trigger "XaphanHelper/MultiMusicTrigger" MultiMusicTrigger(x::Integer, y::Integer, width::Integer=16, height::Integer=16,flagA::String="", flagB::String="", trackNone::String="", trackA::String="", trackB::String="", trackBoth::String="")

const placements = Ahorn.PlacementDict(
    "Multi Music Trigger (Xaphan Helper)" => Ahorn.EntityPlacement(
        MultiMusicTrigger,
        "rectangle"
    )
)

end